doc/import-block-pv.md

```diff
@@ -0,0 +1,120 @@
+# How to import an image to a block raw PV User Guide
+The purpose of this document is to show how to import an image to a raw block PV.
+For now, this functionality only works for importing an image from a URL (http/s3) without authentication.
+
+## Prerequesites
+- You have a Kubernetes cluster up and running with CDI installed and at least one PersistentVolume is available.
+- Feature-Gate 'BlockVolume' is enabled.
+
+## Import an image to a Local raw block PV
+In case you do not import an image to a local PV, please skip to the next section (Import an image with DataVolume manifest).
+There are few steps to follow, to be able to import an image to a local raw block PV.
+
+First, create a local PV with the volumeMode field sets to 'Block':
+
+```bash
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: local-block-raw-import-pv
+  annotations:
+spec:
+  nodeAffinity:
+    required:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: kubernetes.io/hostname
+          operator: In
+          values:
+          - node01
+  volumeMode: Block  # This is Block PV
+  storageClassName: local
+  capacity:
+    storage: 1Gi
+  local:
+    path: /dev/loop0   # This is the local path on the node where we import the image to
+  accessModes:
+    - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Delete 
+```
+
+Then, create the local block device on the node itself (in that case node01 as defined in the PV nodeAffinity), by running the following commands as root on the node:
+```bash
+dd if=/dev/zero of=loopbackfile0 bs=100M count=20
+```
+```bash
+losetup /dev/loop0 loopbackfile0
+```
+
+Note: the path you create on the node has to be the same path that you define in the PV.
+
+
+
+## Import an image with DataVolume manifest
+
+Create the following DataVolume manifest (import-block-pv-datavolume.yaml):
+
+```bash
+apiVersion: cdi.kubevirt.io/v1alpha1
+kind: DataVolume
+metadata:
+  name: import-block-pv-datavolume
+spec:
+  # Optional: Set the storage class or omit to accept the default
+  storageClassName: local
+  volumeMode: Block
```

Comment:
please add this field to validation part in data volume CR definition (and also to validating webhook as Alexander suggested).